### PR TITLE
Add robot names

### PR DESCRIPTION
**Pull request by anar4732**

### CODE REVIEW

File: src/main/resources/assets/opencomputers/robot.names [hunk 1]
```diff
@@ -13,6 +13,7 @@
 Adventure Core     # Portal
 Adjutant 23-46     # Starcraft 2
 Anson Argyris      # Perry Rhodan
+ARR                # Simple
```
Comment (payonel) [inline src/main/resources/assets/opencomputers/robot.names]: do you have a link to this robot? i like to do spot checks on robot names and this one i didn't find anything about

File: src/main/resources/assets/opencomputers/robot.names [hunk 2]
```diff
@@ -77,6 +78,7 @@ Mawhrin-skel       # The Player of Games (Iain M Banks)
 Michiyo            # Contributor
 *Mute              # Analogue: A Hate Story / Hate Plus
 Mycroft Holmes     # The Moon Is a Harsh Mistress
+PacMan             # Pacman
```
Comment (payonel) [inline src/main/resources/assets/opencomputers/robot.names:87]: i wouldn't consider pacman a robot

File: src/main/resources/assets/opencomputers/robot.names [hunk 3]
```diff
@@ -86,6 +88,8 @@ R. Giskard Reventlov # Isaac Asimov's
 R2-D2              # Star Wars
 Replicator         # Stargate
 Robby              # Forbidden Planet
+Robot              # Robot
+RO-BOT             # Robot
```
Comment (payonel) [inline src/main/resources/assets/opencomputers/robot.names]: what movie or book is "Robot"? and are "Robot" and "RO-BOT" names of robots?
Though, having just "robot" as a possible name might be amusing

File: src/main/resources/assets/opencomputers/robot.names [hunk 4]
```diff
@@ -94,6 +98,7 @@ SHODAN             # System Shock
 Skynet             # Terminator
 Space Core         # Portal
 SpiritedDusty      # Contributor
+T.A.R.S            # Interstellar
```
Comment (payonel) [inline src/main/resources/assets/opencomputers/robot.names]: Excellent suggestion!

**Comment by elTiempoVuela** [comment]
YOU ARE A ROBOT?

File: src/main/resources/assets/opencomputers/robot.names (hunk 1 above)
Comment (elTiempoVuela) [reply src/main/resources/assets/opencomputers/robot.names]: VisiionRemote

File: src/main/resources/assets/opencomputers/robot.names (hunk 3 above)
Comment (anar4732) [reply src/main/resources/assets/opencomputers/robot.names]: No, it's just a nice name.

**Pull request merged**